### PR TITLE
Created exponential_search.c

### PR DESCRIPTION
**Pull request by sidntrivedi**

### CODE REVIEW

File: search-algos/exponential_search/exponential_search.c [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+int min(int i,int j)
+{
+    if(j>i)
+         return i;
+    else
+         return j;
+}
+int binary_search(int number,int beg,int end)   /*binary search function implementing its algo and
+{                                                   finding element in the given range.
```
Comment (khanchi97) [inline search-algos/exponential_search/exponential_search.c:22]: why this code is commented?

Comment (sidntrivedi) [reply search-algos/exponential_search/exponential_search.c:22]: Sorry,added comments afterwards.Forgot to close.Updating it now.

**Comment by 22shubh22** [comment]
@sidntrivedi012 please close this PR. As https://github.com/khanchi97/Smvdu-Algos/pull/161 is already open.

**Comment by sidntrivedi** [comment]
Ok Sir.

**Pull request closed without merging**